frontend/components/events/EventCard.tsx

```diff
@@ -19,71 +19,65 @@ interface EventCardProps {
-                )}
-              </div>
-              <div className="mt-4 mb-7 space-y-3">
-                <div className="flex items-center">
-                  <MapPinIcon className="w-5" />
-                  <p className="ml-1 font-light text-sm whitespace-nowrap overflow-hidden">{props.location}</p>
-                </div>
-                <div className="flex items-center">
-                  <CurrencyDollarIcon className="w-5" />
-                  <p className="ml-1 font-light text-sm">{`$${props.price} AUD per person`}</p>
-                </div>
+  const cardContent = (
+    <div className="bg-white rounded-lg text-left border-gray-300 border w-full md:w-[300px] xl:w-[290px] 2xl:w-[320px] hover:cursor-pointer overflow-hidden">
+      {loading ? (
+        <div>
+          <LoadingSkeletonEventCard />
+        </div>
+      ) : (
+        <>
+          <div
+            className="h-36 w-full"
+            style={{
+              backgroundImage: `url(${image})`,
+              backgroundSize: "cover",
+              backgroundPosition: "center center",
+            }}
+          ></div>
+          <div className="p-4">
+            <h4 className="font-bold text-gray-500 text-xs">{timestampToEventCardDateString(startTime)}</h4>
+            <h2 className="text-xl font-bold mb-1 mt-1 whitespace-nowrap overflow-hidden">{name}</h2>
+            <div className="flex ml-0.5 items-center">
+              <Image src={organiser.profilePicture} alt="DP" width={50} height={50} className="rounded-full w-4 h-4" />
+              <p className="text-xs font-light ml-1">{`Hosted by ${organiser.firstName} ${organiser.surname}`}</p>
+              {organiser.isVerifiedOrganiser && <Image src={Tick} alt="Verified Organiser" className="h-4 w-4 ml-1" />}
+            </div>
+            <div className="mt-4 mb-7 space-y-3">
+              <div className="flex items-center">
+                <MapPinIcon className="w-5" />
+                <p className="ml-1 font-light text-sm whitespace-nowrap overflow-hidden">{location}</p>
               </div>
               <div className="flex items-center">
-                <p className="text-sm font-light text-gray-500">{`${props.vacancy} spots left`}</p>
-                {/* <button className="ml-auto rounded-lg border-black border py-1 px-2 text-blackm text-center">
-                  <h2 className="text-sm">Book Now</h2>
-                </button> */}
+                <CurrencyDollarIcon className="w-5" />
+                <p className="ml-1 font-light text-sm">{`$${price} AUD per person`}</p>
               </div>
             </div>
-          </>
-        )}
-      </div>
-    </Link>
+            <div className="flex items-center">
+              <p className="text-sm font-light text-gray-500">{`${vacancy} spots left`}</p>
+            </div>
+          </div>
+        </>
+      )}
+    </div>
   );
+
+  return isPreview ? cardContent : <Link href={`/event/${eventId}`}>{cardContent}</Link>;
```

Comment:
Instead of doing this way, is it possible to simple make the next/Link not clickable? Is there a property that lets us do that?

Reply:
i dont think its possible